odpf/optimus/plugins/v1beta1/dependency_resolver.proto

```diff
@@ -21,10 +21,17 @@ service DependencyResolverModService {
     rpc GenerateDependencies(GenerateDependenciesRequest) returns (GenerateDependenciesResponse);
 }
 
+message PluginJobConfigs {
+    repeated Config configs = 1;
+    repeated Config secrets = 2;
+}
+
 message GenerateDestinationRequest {
-    Configs config = 1;
-    Assets assets = 2;
-    odpf.optimus.core.v1beta1.ProjectSpecification project = 3;
+    Configs config = 1  [deprecated=true];
+    Assets assets = 2  [deprecated=true];
+    odpf.optimus.core.v1beta1.ProjectSpecification project = 3  [deprecated=true];
+
+    PluginJobConfigs job_configs = 4;
```

Comment:
JobConfig & Assets we need to pass and we are passing those details, we don't need to deprecate assets.
PluginJobConfigs is not necessary imo we can leverage configs itself